src/org/zaproxy/zap/extension/jython/ExtensionJython.java

```diff
@@ -50,11 +53,24 @@ public ExtensionJython() {
 	public void hook(ExtensionHook extensionHook) {
 		super.hook(extensionHook);
 
+		this.jythonOptionParam = new JythonOptionParam();
+		extensionHook.addOptionsParamSet(this.jythonOptionParam);
+		if(View.isInitialised()){
+			extensionHook.getHookView().addOptionPanel(new JythonOptionPanel());
+		}
+	}
+	
+	@Override
+	public void start() {
+		super.start();
 
-		ScriptEngineManager mgr = new ScriptEngineManager();
+		String modulePath = this.jythonOptionParam.getModulePath();
+		if (false == Strings.isNullOrEmpty(modulePath)) {

```

Comment:
It could just use/negate the return value (in any case, throughout the code, the values are placed in the right side of the expression).
Same comment for other if statements.